torch/fx/experimental/proxy_tensor.py

```diff
@@ -657,7 +674,7 @@ def disable_autocast_cache():
         torch.set_autocast_cache_enabled(old_value)
 
 
-def make_fx(f, decomposition_table=None, tracing_mode="real", _allow_non_fake_inputs=False):
+def make_fx(f, decomposition_table=None, tracing_mode="real", _allow_non_fake_inputs=False, pre_autograd=False):
```

Comment:
Make this kwarg only

Reply:
Sounds good. At first, I was wary of making mildly BC-breaking schema changes to make_fx. But I suppose it's private, since we don't really have docs on it.

Reply:
I'm probably going to avoid doing that in this PR then just to decouple the risky change

Reply:
Sorry, why can't `pre_autograd` only be kwarg only lol

Reply:
Sorry, should have made it a kwarg - at first I thought you were calling out that all of these inputs should be kwargs, which seems right but also will probably break internal.

Reply:
fixed at https://github.com/pytorch/pytorch/pull/97559